docker/depends/Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN apt-get update \
 # ----------------------------------------------------------------------
 COPY pecan.depends.R /
 RUN  Rscript -e "install.packages(c('devtools'))" \
-  && Rscript -e "devtools::install_version('roxygen2', '7.0.2')" \
```

Comment:
I think we need to update https://github.com/PecanProject/pecan/blob/develop/Makefile#L81 that is the check to make sure the right roxygen is installed.

Reply:
That's updated as part of #2890 -- if we change it here then the checks will fail